index.bs

```diff
@@ -1213,6 +1209,52 @@ Advisory Board and Technical Architecture Group Elections</h5>
 	Refer to <a href="https://www.w3.org/2002/10/election-howto">How to Organize an Advisory Board or TAG election</a> [[ELECTION-HOWTO]]
 	for more details.
 
+<h5 id="TAG-appointments">
+Technical Architecture Group Appointments</h5>
+
+	The [=Team=] is responsible for appointing
+	3 of the participants to the [=Technical Architecture Group=].
+	This mechanism complements the election process.
+	The [=Team=] <em class=rfc2119>should</em> use its appointments to support
+	a diverse and well-balanced [=TAG=],
+	including diversity of technical background, knowledge, and skill sets.
+
+	The [=Team=] <em class=rfc2119>should</em> actively seek
+	candidates for appointment to the TAG,
+	and <em class=rfc2119>must</em> make available to
+	the W3C community at large
+	a means to propose candidates for consideration,
+	explicitly soliciting input from at least
+	current and incoming [=TAG=] members, the [=Advisory Committee=], and [=Working Group=] [=Chairs=].
+
+	The constraints for appointment to the [=TAG=] are
+	the same as for elected participants
+	(see [[#AB-TAG-constraints]] and [[#AB-TAG-elections]]),
+	with the additional constraint that
+	a person <em class=rfc2119>must not</em> be appointed
+	for more than two <em>consecutive</em> terms.
+	(Partial terms used to fill a vacated seat do not count towards this limit.)
+
+	Note: Individuals who have reached the limit of two consecutive appointed terms
+	<em class=rfc2119>may</em> freely run for election
+	if they wish to continue serving on the [=TAG=].
+
+	The [=Team=]'s choice of appointee(s)
+	is subject to ratification by secret ballot
+	by two thirds of the [=TAG=].
+	In the case of regularly scheduled election,
+	ratification is by the [=TAG=] members of the upcoming term.
```

Comment:
Should it be explicit that ratification takes place_before_ announcement?

Reply:
I think it is already? In the next paragraph: “the Team should announce the *ratified* appointment […]”